sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -888,6 +893,7 @@ public partial class HttpPipeline
         public static System.IDisposable CreateHttpMessagePropertiesScope(System.Collections.Generic.IDictionary<string, object?> messageProperties) { throw null; }
         public Azure.Core.HttpMessage CreateMessage() { throw null; }
         public Azure.Core.HttpMessage CreateMessage(Azure.RequestContext? context) { throw null; }
+        public Azure.Core.HttpMessage CreateMessage(Azure.RequestContext? context, Azure.Core.CoreResponseClassifier? classifier = null) { throw null; }
```

Comment:
Can this take the base class as a parameter?

Reply:
Yeah, we would need to do develop internal "ChainedClassifier". i.e. we would take ResponseClassifier, dynamically check if it's CoreResponseClassifier. If it is, we would clone. If it's not, we would wrap the ResponseClassifier and all RequestContext data in the ChainedClassifier. 

I think it's OK to punt this for now (this PR) but we should think about doing it before we GA. The reason is that I suspect we will need this anyway, and if we GA the current overload, we would endup with one redundant overload.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/27104